src/background/services/paymentManager.ts

```diff
@@ -202,6 +201,7 @@ export class PaymentManager {
     payableSessions: PaymentSession[],
     signal?: AbortSignal,
   ) {
+    const { isOutOfBalanceError } = await import('./openPayments');
```

Comment:
Direct import was having issue as well with testing - similar to some other modules.